frontend/src/scenes/notebooks/Nodes/NotebookNodeQuery.tsx

```diff
@@ -105,13 +104,22 @@ export const Settings = ({
         const modifiedQuery = { ...query, full: false }
 
         if (NodeKind.DataTableNode === modifiedQuery.kind || NodeKind.SavedInsightNode === modifiedQuery.kind) {
-            // We don't want to show the insights button for now
             modifiedQuery.showOpenEditorButton = false
             modifiedQuery.showHogQLEditor = true
             modifiedQuery.showResultsTable = false
-            modifiedQuery.showReload = false
-            modifiedQuery.showElapsedTime = false
+
+            modifiedQuery.showReload = true
+            modifiedQuery.showElapsedTime = true
+            modifiedQuery.showTimings = true
```

Comment:
These three properties are causing my browser to crash. Something seems to be infinitely reloading the insight